extensions/amp-analytics/0.1/config.js

```diff
@@ -64,11 +66,63 @@ export class AnalyticsConfig {
     this.win_ = this.element_.ownerDocument.defaultView;
     this.isSandbox_ = this.element_.hasAttribute('sandbox');
 
-    return this.fetchRemoteConfig_()
+    return Promise.all([this.fetchRemoteConfig_(), this.fetchVendorConfig_()])
       .then(this.processConfigs_.bind(this))
       .then(() => this.config_);
   }
 
+  /**
+   * Constructs the URL where the given vendor config is located
+   * @private
+   * @param {string} vendor the vendor name
+   * @return {string} the URL to request the vendor config file from
+   */
+  getVendorUrl_(vendor) {
+    const isLocalDev = getMode().localDev;
+    const baseUrl = calculateScriptBaseUrl(this.win_.location, isLocalDev);
+    const rtv = isLocalDev ? '' : `/rtv/${getMode().rtvVersion}`;
```

Comment:
Are you planning to include the local server change in a separate PR? 
nit: we can remove the isLocalDev rtv calculation as well. So that local server always expect a request to rtv/12345 but rewrite it later. Thanks.

Reply:
done :)